drivers/usb/device/usb_dc_sam0.c

```diff
@@ -132,6 +136,15 @@ static void usb_sam0_isr(void)
 		data->cb(USB_DC_RESET, NULL);
 	}
 
+	/* Suspend interrupt */
+	if ((intflag & USB_DEVICE_INTFLAG_SUSPEND) != 0U) {
+		/* Disable suspend interrupt */
```

Comment:
nit: the comment doesn't add any new information.  Change to why the interrupt is disabled at this point or drop.

Reply:
It will be handled even if the suspend interrupt is disabled. And yes please help to save the planet remove unnecessary comments (less storage consumption).

Reply:
> It will be handled even if the suspend interrupt is disabled.

My observation is that the suspend interrupt still occurs even if it is disabled **when the device is re-plugged**; but, in order to get the suspend (as in detach) interrupt when you actually unplug, you must enable this interrupt here.

> please help to save the planet remove unnecessary comments (less storage consumption).

You must be very environmentally conscious. Are you powering your workstation with a bike generator too? :)